apps/admin/frontend/src/screens/tally/remove_all_cvrs_modal.tsx

```diff
@@ -22,58 +22,51 @@ export function ConfirmRemoveCvrsModal({
 
   if (
     !castVoteRecordFilesQuery.isSuccess ||
-    !manualDataResultsMetadataQuery.isSuccess ||
+    !manualResultsMetadataQuery.isSuccess ||
     !castVoteRecordFileModeQuery.isSuccess
   ) {
     return null;
   }
 
-  const hasManualData = manualDataResultsMetadataQuery.data.length > 0;
+  const hasManualResults = manualResultsMetadataQuery.data.length > 0;
 
-  function removeCvrs() {
+  function removeCvrs({ doCloseModal }: { doCloseModal: boolean }) {
     clearCastVoteRecordFilesMutation.mutate(undefined, {
-      onSuccess: onClose,
+      onSuccess: doCloseModal ? onClose : undefined,
     });
   }
 
-  async function removeCvrsAndManualResults() {
-    await Promise.all([
-      clearCastVoteRecordFilesMutation.mutateAsync(),
-      deleteAllManualResultsMutation.mutateAsync(),
-    ]);
-    onClose();
+  function removeManualResults() {
+    deleteAllManualResultsMutation.mutate(undefined, {
+      onSuccess: onClose,
+    });
   }
 
-  if (hasManualData) {
-    const anyMutationIsLoading =
-      clearCastVoteRecordFilesMutation.isLoading ||
-      deleteAllManualResultsMutation.isLoading;
+  if (!clearCastVoteRecordFilesMutation.isSuccess) {
```

Comment:
Did you ensure that in the case where there were no manual tallies, it closes the modal before `isSuccess` is true (which could briefly flash the manual tallies modal)?

Reply:
It doesn't flash (I looked at a frame-by-frame) in my testing. This is why I was asking you that unintelligible question about mutation state yesterday, to understand if this would be an issue.